sqjobs/connectors/dummy.py

```diff
@@ -18,6 +18,12 @@ def __init__(self):
     def get_queue(self, name):
         return self.jobs.setdefault(name, [])
 
+    def get_queues(self):
+        return self.jobs.keys()
```

Comment:
Force this to be a list to be consistent with the behaviour of the SQS connector